src/main/java/org/folio/services/impl/PasswordStorageServiceImpl.java

```diff
@@ -484,17 +483,17 @@ public PasswordStorageService isPasswordPreviouslyUsed(JsonObject passwordJson,
   private Future<Credential> updateCred(AsyncResult<SQLConnection> conn, String tenantId,
                                         Credential newCred, Credential oldCred) {
     PostgresClient pgClient = PostgresClient.getInstance(vertx, tenantId);
-    Future<UpdateResult> future = Future.future();
+    Promise<Credential> promise = Promise.promise();
     CQL2PgJSON field = null;
     try {
       field = new CQL2PgJSON(TABLE_NAME_CREDENTIALS + ".jsonb");
     } catch (FieldException e) {
-      future.fail(e);
+      promise.fail(e);
     }
     CQLWrapper cqlWrapper = new CQLWrapper(field, "id==" + newCred.getId());
-    pgClient.update(conn, TABLE_NAME_CREDENTIALS, newCred, cqlWrapper, true, future.completer());
+    pgClient.update(conn, TABLE_NAME_CREDENTIALS, newCred, cqlWrapper, true, reply -> promise.complete(oldCred));
```

Comment:
`future.completer()` is exactly the same as `future`: https://github.com/eclipse-vertx/vert.x/blob/3.9.1/src/main/java/io/vertx/core/Future.java#L471-L479
`future` expressed using promise is `promise.future()`.

`reply -> promise.complete(oldCred)` is wrong because it incorrectly drops the exception on failure.

Please also fix all other `future.completer()` replacements.